components/infobox/commons/infobox_league.lua

```diff
@@ -237,7 +290,15 @@ function League:_definePageVariables(args)
 	Variables.varDefine('tournament_series', mw.ext.TeamLiquidIntegration.resolve_redirect(args.series or ''))
 
 	Variables.varDefine('tournament_liquipediatier', args.liquipediatier)
-	Variables.varDefine('tournament_liquipediatiertype', args.liquipediatiertype)
+	Variables.varDefine(
+		'tournament_liquipediatiertype',
+		Tier.text.types
+			and Tier.text.types[args.liquipediatiertype or '']
+			or args.liquipediatiertype
```

Comment:
Wouldn't this force the input to be in lowercase, compared to the Upperfirst that's used right now?

Reply:
no,

currently we store the input as entered
with this change:
- if the tier module is updated it get the to be stored value from that table and hence capitalisation is standardized if it is a valid tier type
--> if the entered tiertype is invalid atm it will still (as before) store the invalid entry into the wiki var
--> the input into the `Tier.text.types` table has to be lower cased though, adjusted that now
- if the tier module is not yet updated it will store as before just the plain input